test/gtest/conv_embed_db.cpp

```diff
@@ -0,0 +1,243 @@
+#include <tuple>
+
+#include <miopen/miopen.h>
+#include <gtest/gtest.h>
+#include "conv_2d.hpp"
+#include "get_handle.hpp"
+
+using TestCase = std::tuple<std::vector<std::string>, std::string>;
+
+enum class Precision
+{
+    Float,
+    Half,
+    Int8,
+    BFloat16
+};
+
+std::string GetFloatArg()
+{
+    static const auto tmp = std::getenv("MIOPEN_TEST_FLOAT_ARG");
+    if(tmp == nullptr)
+    {
+        return "";
+    }
+    return tmp;
+};
```

Comment:
Reuse stuff from <miopen/env.hpp>

Reply:
@atamazov , GetEnvVars() takes const std::vector<std::string>& as input, and append "=0" at the end of each env var that could be found. From what I can see, the only reusable code in env.hpp is miopen::GetEnv() to replace std::getenv() here